ts/twitter/src/app/notifications/_components/notifications.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Box, Flex } from "@chakra-ui/react";
+import { IoIosSettings } from "react-icons/io";
+import { SettingButton } from "@/lib/components/setting-button";
+import { NotificationsTabBar } from "./notifications-tab-bar";
+
+export const Notifications: React.FC = () => {
+  return (
+    <Box>
+      <Flex>
+        <Box mx="12px" mt="8px" gap="4" fontSize="lg">
+          Notifications
+        </Box>
+        <SettingButton
+          tooltipText={"setting"}
+          ariaLabel={"setting"}
```

Comment:
Let's change 'setting' to 'settings' since 'settings' is used in the actual X and sounds more natural.

Reference: the settings button in the actual X, placed in the upper middle right of the view.
![Screenshot 2024-12-03 at 10 10 26 PM (2)](https://github.com/user-attachments/assets/5af54359-baef-4416-8f53-4db92eaa6161)

```suggestion
          tooltipText={"settings"}
          ariaLabel={"settings"}
```